news/11096.feature.rst

```diff
@@ -0,0 +1 @@
+Add ``--dry-run`` to pip install.
```

Comment:
This should probably very briefly describe what the option does?

Reply:
I added some more details.